sqe-database-access/DbConnectionBase.cs

```diff
@@ -390,15 +345,15 @@ public override CommandType CommandType
             set => _underlyingSqlCommand.CommandType = value;
         }
 
-        protected override DbConnection DbConnection
+        protected sealed override DbConnection DbConnection
```

Comment:
Why seal ir? I'm actually not crazy about sealed (or Java's final) at all. I never did understand what it's good for.

Reply:
I gave in to the peer pressure of my IDE.  I also hate sealed, it feels simply too bureaucratic. I will fix that in the next PR.